routes/users.js

```diff
@@ -1,4 +1,16 @@
 const express = require('express');
 const router = express.Router();
+let usersJson = require('../data/users.json');
+
+router.get('/', (req, res, next) => {
+    res.send(usersJson);    
+    });
+  
+    router.get('/:id', (req, res, next) => {
```

Comment:
This should be at the same indent level ast the other router.get definition. 
No penalty, just a clean-coding note. At first glance, the second route appears nested inside the first.